libs/transfer/src/lib/transfer-journey.component.html

```diff
@@ -1,3 +1,12 @@
+<div class="row" *ngIf="counterPartyName">
+  <bb-alert-ui
+    title="Transfer Alert"
+    modifier="info"
+    [dismissible]="true"
+    [message]="'Making Repeated Transfer for ' + counterPartyName"
```

Comment:
Can we add translations for that?

Reply:
And for the title?

Reply:
I have created a separate issue for this to add translations.